{"filter":false,"title":"my_package_launch_file.launch","tooltip":"/my_package/launch/my_package_launch_file.launch","undoManager":{"mark":0,"position":0,"stack":[[{"start":{"row":0,"column":0},"end":{"row":4,"column":9},"action":"insert","lines":["<launch>","    <!-- My Package launch file -->","    <node pkg=\"my_package\" type=\"simple.py\" name=\"ObiWan\"  output=\"screen\">","    </node>","</launch>"],"id":1}]]},"ace":{"folds":[],"scrolltop":0,"scrollleft":0,"selection":{"start":{"row":0,"column":0},"end":{"row":4,"column":9},"isBackwards":true},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"timestamp":1547095798000,"hash":"0d8dc671120f26b254dd110cb6f3482d5e298d9a"}